assets/vendor/bootstrap/scss/_variables.scss

```diff
@@ -715,7 +715,7 @@ $nav-divider-margin-y:              $spacer / 2 !default;
 
 // Navbar
 
-$navbar-padding-y:                  $spacer / 2 !default;
+$navbar-padding-y:                  math.div($spacer, 2);
```

Comment:
Replacing only with `math.div()` might result in the following error.

<img width="571" alt="image" src="https://user-images.githubusercontent.com/58922834/224858968-08952066-8fca-4272-9380-a417d9d3d8b2.png">

It is recommended to build and test locally.